lib/common.sh

```diff
@@ -208,7 +208,9 @@ fi
 OS=$(awk -F= '/^ID=/ { print $2 }' /etc/os-release | tr -d '"')
 export OS
 OS_VERSION=$(awk -F= '/^VERSION_ID=/ { print $2 }' /etc/os-release | tr -d '"' | cut -f1 -d'.')
+OS_VERSION_ID=$(awk -F= '/^VERSION_ID=/ { print $2 }' /etc/os-release | tr -d '"')
```

Comment:
OS_VERSION=$(awk -F= '/^VERSION_ID=/ { print $2 }' /etc/os-release | tr -d '"')  and remove OS_VERSION_ID

I do not see the relevance of having both 18 and 18.04 as the comparison can be made string based.

Reply:
OS_VERSION is used for CentOS, OS_VERSION_ID is used for Ubuntu, because we need a specific minor version, but in CentOS we have to stick with the major

Reply:
Got it.